config-connector/solutions/sql/helm/postgres-ha/README.md

```diff
@@ -0,0 +1,93 @@
+# PostgreSQL High Availability
+
+==================================================
+
+## NAME
+
+  postgres-ha
+
+## SYNOPSIS
+
+  Config Connector compatible yaml files to configure a high availability PostgreSQL cluster.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the service account folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/sql/helm/postgres-ha
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](../../../README.md#helm)
+1. A working Config Connector instance.
+1. Cloud SQL Admin API enabled in the project where Config Connector is
+      installed
+1. Cloud SQL Admin API enabled in the project managed by Config Connector if
+      it is a different project
+1. The "cnrm-system" service account with either both `roles/cloudsql.admin` or `roles/owner` in the project managed by Config Connector.
+
+## USAGE
+
+All steps are run from the current directory ([config-connector/solutions/sql/helm/postgres-ha](.)).
+
+1. Review and update the values in `./values.yaml`.
+
+1. Configure a high availability PostgreSQL cluster with Helm.
+
+    ```bash
+    # validate your chart
+    helm lint . --set User1.Name=first-username,User2.Name=second-username,User3.Name=third-username,User1.Password=$(echo -n 'first-password' | base64),User2.Password=$(echo -n 'second-password' | base64),User3.Password=$(echo -n 'third-password' | base64)
```

Comment:
Setting the values in the command like this takes up a lot of space in the doc with so many values, and shouldn't be necessary if the user set the values appropriately in the first step. It might be good to let the user know that the password fields should be base64, but I wouldn't otherwise include any of the `--set` portion of this and following commands.

Reply:
I kinda disagree.
Taking up a lot of space in the doc with so many values shouldn't be an issue as long as users can simply copy, paste, and run the command. We should provide a consistent user experience within the all readmes across the solutions. AFAIK all the other readmes set required field using `--set` so having it here as well seems to be very reasonable.

Reply:
splitting the single line command into multiple lines so that it is easy to see.